keylime.conf

```diff
@@ -312,7 +312,7 @@ max_retries = 10
 # might provide a signed list of EK public key hashes.  Then you could write
 # an ek_check_script that checks the signature of the whitelist and then
 # compares the hash of the given EK with the whistlist
-require_ek_cert = True
+require_ek_cert = False
```

Comment:
Same again, needs to stay as `True` or it creates a security hole for anyone using a Hardware TPM

Reply:
We can still solve this with `keylime/keylime/provider_platform_init.py` above during developing right?

Reply:
Honestly, its not a big problem and its fine being set this way in this branch, my main point was we can't have `False` as the default for `master` branch and a release.

Reply:
Got it, understand.